src/cocoa/BugsnagCocos2dxPlugin.m

```diff
@@ -1,92 +0,0 @@
-#import <BugsnagStatic/Bugsnag.h>
-#import <BugsnagStatic/BugsnagNotifier.h>
-#import <BugsnagStatic/BugsnagPlugin.h>
-
-#import "BugsnagCocos2dxPlugin.h"
-
-NSString *const BUGSNAG_COCOS2DX_VERSION = @"1.1.0";
-
-@interface Bugsnag ()
-+ (BugsnagNotifier *)notifier;
-+ (void)registerPlugin:(id<BugsnagPlugin>)plugin;
-@end
-
-@interface BugsnagCocos2dxPlugin () <BugsnagPlugin>
-@property(nonatomic, getter=isStarted) BOOL started;
-@property(nonatomic) NSString *cocos2dVersion;
-@end
-
-@implementation BugsnagCocos2dxPlugin
-
-+ (void)registerWithCocos2dVersion:(const char *)rawVersion {
-  BugsnagCocos2dxPlugin *plugin = [BugsnagCocos2dxPlugin new];
-  plugin.cocos2dVersion = [self parseCocos2dVersion:rawVersion];
-  [Bugsnag registerPlugin:plugin];
-}
-
-+ (NSString *)parseCocos2dVersion:(const char *)rawVersion {
-    // cocos2d::cocos2dVersion() is in the format "cocos2d-x-3.1.0" or
-    // "cocos2d-x 3.17.0", so we make some small adjustments to correctly format
-    // the value to a semantic version.
-    NSString *ns_rawVersion =
-        rawVersion == NULL ? nil
-                           : [NSString stringWithCString:rawVersion
-                                                encoding:NSUTF8StringEncoding];
-    NSCharacterSet *separators =
-        [NSCharacterSet characterSetWithCharactersInString:@"- "];
-    return [[ns_rawVersion
-        componentsSeparatedByCharactersInSet:separators] lastObject];
-}
-
-- (void)start {
-  [self setNotifierDetails];
-  [self addPayloadModifiers];
-
-  self.started = YES;
-}
-
-/** Set the plugin's notifier info over the default */
-- (void)setNotifierDetails {
-  id notifier = [Bugsnag notifier];
-  NSDictionary *newDetails = @{
-    @"version" : BUGSNAG_COCOS2DX_VERSION,
-    @"name" : @"Bugsnag Cocos2d-x",
-    @"url" : @"https://github.com/bugsnag/bugsnag-cocos2dx"
-  };
-  [notifier setValue:newDetails forKey:@"details"];
-}
```

Comment:
Don't we still need to set the notifier details?

Reply:
yes we do

Reply:
This is now here: https://github.com/bugsnag/bugsnag-cocos2dx/pull/12/files#diff-4e086e6e53940cd09dfbc58fb7540d03ca44777325e573bc9b9cb86c3efde076R30